ChallengesWithTestsMarkVII/ChallengesSet01.cs

```diff
@@ -8,37 +8,50 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            return (num1==num2);
```

Comment:
These parentheses (and the ones in the other methods) aren't necessary.  Sometimes they add clarity, but it's a little uncommon in C# to use them all the time.
This could just be `return num1 == num2;`